src/MOI_wrapper.jl

```diff
@@ -319,9 +319,68 @@ function _get_at_index(
     return _get(optimizer, attr, ci_primal, ci_dual)[idx]
 end
 
+_minus(::Nothing) = nothing
+_minus(x) = -x
+
+function _dual_attribute(attr::Union{MOI.VariablePrimal,MOI.ConstraintPrimal})
+    return MOI.ConstraintDual(attr.result_index)
+end
+
+function _dual_attribute(
+    ::Union{MOI.VariablePrimalStart,MOI.ConstraintPrimalStart},
+)
+    return MOI.ConstraintDualStart()
+end
+
+function _dual_attribute(attr::MOI.ConstraintDual)
+    return MOI.ConstraintPrimal(attr.result_index)
+end
+
+function _dual_attribute(::MOI.ConstraintDualStart)
+    return MOI.ConstraintPrimalStart()
+end
+
+function _variable_dual_attribute(attr::MOI.ConstraintDual)
+    return MOI.VariablePrimal(attr.result_index)
+end
+
+function _variable_dual_attribute(::MOI.ConstraintDualStart)
+    return MOI.VariablePrimalStart()
+end
+
+function MOI.supports(
+    ::DualOptimizer,
+    ::MOI.VariablePrimalStart,
+    ::Type{MOI.VariableIndex},
+)
+    return true
```

Comment:
This needs to check whether the inner optimizer supports `ConstraintDualStart`

Reply:
It's a bit tricky because we don't know the type of constraints. The inner may not support equality constraints in which case we cannot even ask for whether the solver supports `ScalarAffineFunction`-in-`EqualTo`

Reply:
Yeah I don't know what a good answer is here

Reply:
I think I have an idea that I'll try next week